productivity-suite/app/legacy-app/src/components/pages/News.tsx

```diff
@@ -11,8 +15,20 @@ export function News() {
     };
   });
 
+  const ref = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (ref.current) {
+      return getBus(ref.current).listen<number>("news-page-set", (newsPage) => {
```

Comment:
Since this is not inside a fragment, you don't actually need a `ref` you can just call `getBus()`. But perhaps that might be confusing?

Reply:
yes.... the fact is that you could never use the ref/element argument and everything would still work fine (except from memory leaks)

so I was sort of thinking that maybe we should always try to use an element whenever possible and only use `getBus()` when there are no elements, that does sound like a sort best-practice-like/of clear cut rule... but I am not sure if that makes too much sense

otherwise I guess the rule here would be to never provide the element and only do so inside fragments?
(PS: but also there, if you only dispatch a value you could as well just not use the element right?)